### ISSUE TITLE
Shared folder is synced only after vibrant reload

### ISSUE THREAD

**Issue by gitusercoder** [hashicorp/vagrant#12083]
I am running the last version of vabrant on a mac. When I reload the vibrant file, the shared folder in the guest machine becomes synchronized with the shared folder on the host machine. But after ssh into the guest machine the shared folder of the guest machine doesn't follow the changes in the shared of the host machine. Also, the shared folder of the host machine never gets synced with the changes in the shared folder of the guest machine. I tried "rsync" and "nfs" and "smb".

**Comment by soapy1** [comment]
Hey there, could you provide a bit more information about this issue. In particular, the relevant parts of your Vagrantfile and [a debug output](https://www.vagrantup.com/docs/other/debugging). Thanks!

**Comment by chrisroberts** [comment]
Hey there,

I am going to close this due to lack of response. If this is still occurring, please open a [_new issue_](https://github.com/hashicorp/vagrant/issues/new) and follow the provided issue template that appears when you click the "New Issue" button. This will help us in getting a reproduction and fix. Thanks! 😄
